bodhi/server/util.py

```diff
@@ -601,7 +601,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
```

Comment:
Since we log stdout at debug, let's change this info to debug as well. No reason to log stderr at info if we're logging stdout at debug.